build gives error if not configured and built since last reboot

### 🐛Bug
when making, an error will be issues if configure has not been run because of a tmp file delted during reboot:
```
CMake Error at App/PRETUS/cmake_install.cmake:69 (file):
  file INSTALL cannot find "/tmp/launcher_pretus.sh": No such file or
  directory.
Call Stack (most recent call first):
  App/cmake_install.cmake:42 (include)
  cmake_install.cmake:45 (include)


make: *** [Makefile:86: install] Error 1
```

### Expected behavior

This error does not happen

--- a/src/App/PRETUS/CMakeLists.txt
+++ b/src/App/PRETUS/CMakeLists.txt
@@ -86,14 +86,14 @@ get_filename_component(PARENT_QT_DIR ${Qt5Core_DIR} DIRECTORY)
 get_filename_component(LOCAL_QT_LIB_DIR ${PARENT_QT_DIR} DIRECTORY)
 get_filename_component(LOCAL_QT_DIR ${LOCAL_QT_LIB_DIR} DIRECTORY)
 
-file(WRITE /tmp/launcher_pretus.sh "#! /bin/bash\n")
-file(APPEND /tmp/launcher_pretus.sh "export QT_PLUGIN_PATH=${LOCAL_QT_DIR}/plugins/\n")
-file(APPEND /tmp/launcher_pretus.sh "export LD_LIBRARY_PATH=${LOCAL_QT_LIB_DIR}/\n")
-file(APPEND /tmp/launcher_pretus.sh "export QTDIR=${LOCAL_QT_DIR}/\n")
-file(APPEND /tmp/launcher_pretus.sh "$PWD/bin/pretus \${\@}\n")
+file(WRITE ${CMAKE_INSTALL_PREFIX}/.tmp/launcher_pretus.sh "#! /bin/bash\n")
+file(APPEND ${CMAKE_INSTALL_PREFIX}/.tmp/launcher_pretus.sh "export QT_PLUGIN_PATH=${LOCAL_QT_DIR}/plugins/\n")
+file(APPEND ${CMAKE_INSTALL_PREFIX}/.tmp/launcher_pretus.sh "export LD_LIBRARY_PATH=${LOCAL_QT_LIB_DIR}/\n")
+file(APPEND ${CMAKE_INSTALL_PREFIX}/.tmp/launcher_pretus.sh "export QTDIR=${LOCAL_QT_DIR}/\n")
+file(APPEND ${CMAKE_INSTALL_PREFIX}/.tmp/launcher_pretus.sh "$PWD/bin/pretus \${\@}\n")
 
 install(
-    FILES /tmp/launcher_pretus.sh
+    FILES ${CMAKE_INSTALL_PREFIX}/.tmp/launcher_pretus.sh
     PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ
     DESTINATION "${CMAKE_INSTALL_PREFIX}"
 )

